examples/raspberrypi/rp2040/build.zig

```diff
@@ -0,0 +1,62 @@
+const std = @import("std");
+const MicroZig = @import("microzig/build");
+const rp2040 = @import("microzig/bsp/raspberrypi/rp2040");
+
+const available_examples = [_]Example{
+    // RaspberryPi Boards:
+    .{ .target = rp2040.boards.raspberrypi.pico, .name = "pico_adc", .file = "src/adc.zig" },
```

Comment:
I'm fine with that change back to what we had before, but i'd add a generic function to add `findTarget` back somehow.
We don't have to expose it as the primary interface, but it makes things easier if we want to pass the target device from the outside